### DISCUSSION TITLE
[Bug Report] Scarpe by url.connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

### DISCUSSION THREAD

**Discussion by codedragonnnnn** [stashapp/stash#4833] [Q&A] [answered]
**Describe the bug**

When I use URL scraping, this problem occurs. I can access this URL in the browser.
I come from China and I use Clash to access the internet

**Desktop (please complete the following information):**
 - OS: Windows11
 - Browser: edge

**Comment by WithoutPants** [comment, marked as answer]
I'm assuming clash is a proxy?

Scraping uses a proxy only if it is configured in the environment. `HTTP_PROXY` or `HTTPS_PROXY` must be set (to the proxy url `https://<proxy_url>:<port>`) in the environment when stash is started in order for stash to use the proxy.
